text/0002-bundle-add-and-remove.md

```diff
@@ -0,0 +1,275 @@
+- Feature Name: bundle\_add\_and\_remove\_improvements
+- Start Date: 2018-02-10
+- RFC PR: (leave this empty)
+- Bundler Issue: (leave this empty)
+
+# Summary
+
+This feature adds improvements for `bundle add` command and adds
+the opposite command for removing gems called `bundle remove`.
+
+The purpose of this RFC is to get feedback on use cases, CLI format/syntax
+and feature options.
+
+# Motivation
+
+The main use case is improving user productivity. 
+
+This is done by adding gem(s) to the Gemfile with the latest version in the range specified from the command line. User saves time since they don't have to search Rubygems for the latest version. 
+
+The opposite command is also available, which provides convenience for removing gem(s) from the Gemfile from the command line.
+
+# Guide-level explanation
+
+The usage of the `add` and `remove` command can be grouped in 7 sections.
+
+## 1. adding gem to group block
+
+### 1.1. when Gemfile doesn't yet have the group specified
+
+#### Proposed command
+
+```ruby
+# Gemfile before
+source "https://rubygems.org"
+```
+
+```ruby
+bundle add minitest --group test
+```
+
+```ruby
+# Gemfile after
+source "https://rubygems.org"
+
+group :test do
+  gem "minitest", "~> 5.11"
+end
+```
+
+### 1.2. when Gemfile has the group specified
+
+#### Proposed command
+
+```ruby
+# Gemfile before
+source "https://rubygems.org"
+
+group :test do
+  gem "minitest", "~> 5.11"
+end
+```
+
+```shell
+bundle add minitest-reporters --group test
+```
+
+```ruby
+# Gemfile after
+source "https://rubygems.org"
+
+group :test do
+  gem "minitest", "~> 5.11"
+  gem "minitest-reporters" "~> 1.1"
+end
+```
+
+### Description
+
+Instead of adding gems to the Gemfile in the format
+
+```ruby
+gem "minitest", "~> 5.11", group: [:test]
+```
+
+the `add` command works with group blocks
+
+
+```ruby
+group :test do
+  gem "minitest", "~> 5.11"
+end
+```
+
+which are more natural for users.
+
+It appends gems to the end of the group block if the group already exists. It also supports adding to multiple groups.
+
+## 2. adding multiple gems with options
+
+#### Proposed command
+
+```shell
+bundle add sinatra:2.0 minitest:~>5.11
```

Comment:
You're going to need quotes around the 2nd gem there, or the shell will interpret the `>` as output redirection. I'd recommend putting single quotes around all the gem names/versions in shell commands you show in the docs.